mobile/library/jni/jni_helper.h

```diff
@@ -218,6 +236,13 @@ class JniHelper {
    */
   [[nodiscard]] LocalRefUniquePtr<jclass> findClass(const char* class_name);
 
+  /**
+   * Finds the given `class_name` from the cache.
+   *
+   * https://docs.oracle.com/en/java/javase/17/docs/specs/jni/functions.html#findclass
+   */
+  [[nodiscard]] jclass findClassFromCache(const char* class_name);
```

Comment:
why isn't this static if `addClassToCache` is static?

Reply:
No strong reason for it not to be `static` other than it's a bit nicer being able to call `jni_helper.findClass` similar like the rest of the code. The `JniHelper` is meant to be a tiny wrapper around JNI API and `findClass` is a function in the JNI API, e.g. (`env->FindClass`) so having `jni_helper.findClass` makes it more consistent with the JNI API.

In the next PR, I plan to remove `findClassFromCache` and simply replace `findClass` implementation with the cache implementation, so the existing code doesn't need to change.